pkg/agent/util/net_linux.go

```diff
@@ -453,6 +454,12 @@ func PrepareHostInterfaceConnection(
 		return "", false, fmt.Errorf("failed to set link up: %v", err)
 	}
 
+	if mtu > 0 {
```

Comment:
Maybe update the comment of this function about the potential MTU changes,

Reply:
done